### ISSUE TITLE
CSFML 2.6.0 doesn't export sfTexture_setSrgb

### ISSUE THREAD

**Issue by racket-raccoon** [massung/r-cade#36]
**TLDR; Removing**
```
(define-sfml sfTexture_setSrgb
  (_fun _sfTexture* _bool -> _void))
```
**from graphics.rkt helps**

I was trying to run the Hello world from the [tutorials page](https://r-cade.io/tutorials) on Windows and faced this error.

```
ffi-obj: could not find export from foreign library
  name: sfTexture_setSrgb
  library: csfml-graphics-2.dll
  system error: The specified procedure could not be found.; win_err=127
  context...:
   C:\Program Files\Racket\collects\ffi\unsafe.rkt:258:20
   C:\Program Files\Racket\collects\ffi\unsafe.rkt:252:2: get-ffi-obj*
   body of "C:\Users\lv\AppData\Roaming\Racket\8.12\pkgs\csfml\graphics.rkt"
```

I was trying random different versions of [CSFML](https://github.com/SFML/CSFML/releases/tag/2.6.0) but couldn't find the one that is fully compatible with the Racket implementation. Just removing sfTexture_setSrgb from the graphics.rkt was enough to run the application with the latest availabel CSFML. Not sure if anyone is up to sort this stuff out.

<details>
  <summary>Full list of CSFML 2.6.0 exports on my machine</summary>
  
  ```
          1    0 00132000 sfBlack
          2    1 000E86F8 sfBlendAdd
          3    2 000E86E0 sfBlendAlpha
          4    3 000E8740 sfBlendMax
          5    4 000E8728 sfBlendMin
          6    5 000E8710 sfBlendMultiply
          7    6 000E8758 sfBlendNone
          8    7 00132010 sfBlue
          9    8 00001970 sfCircleShape_copy
         10    9 00001B70 sfCircleShape_create
         11    A 00001BC0 sfCircleShape_destroy
         12    B 00001BF0 sfCircleShape_getFillColor
         13    C 00001C30 sfCircleShape_getGlobalBounds
         14    D 00001C70 sfCircleShape_getInverseTransform
         15    E 00001D50 sfCircleShape_getLocalBounds
         16    F 00001D90 sfCircleShape_getOrigin
         17   10 00001DC0 sfCircleShape_getOutlineColor
         18   11 00001E00 sfCircleShape_getOutlineThickness
         19   12 00001E10 sfCircleShape_getPoint
         20   13 00001E40 sfCircleShape_getPointCount
         21   14 00001E50 sfCircleShape_getPosition
         22   15 00001E80 sfCircleShape_getRadius
         23   16 00001E90 sfCircleShape_getRotation
         24   17 00001EA0 sfCircleShape_getScale
         25   18 00001ED0 sfCircleShape_getTexture
         26   19 00001EE0 sfCircleShape_getTextureRect
         27   1A 00001F30 sfCircleShape_getTransform
         28   1B 00002010 sfCircleShape_move
         29   1C 00002040 sfCircleShape_rotate
         30   1D 00002050 sfCircleShape_scale
         31   1E 00002080 sfCircleShape_setFillColor
         32   1F 000020C0 sfCircleShape_setOrigin
         33   20 000020F0 sfCircleShape_setOutlineColor
         34   21 00002130 sfCircleShape_setOutlineThickness
         35   22 00002140 sfCircleShape_setPointCount
         36   23 00002150 sfCircleShape_setPosition
         37   24 00002180 sfCircleShape_setRadius
         38   25 00002190 sfCircleShape_setRotation
         39   26 000021A0 sfCircleShape_setScale
         40   27 000021D0 sfCircleShape_setTexture
         41   28 00002210 sfCircleShape_setTextureRect
         42   29 00002240 sfColor_add
         43   2A 000022D0 sfColor_fromInteger
         44   2B 00002300 sfColor_fromRGB
         45   2C 00002320 sfColor_fromRGBA
         46   2D 00002340 sfColor_modulate
         47   2E 000023E0 sfColor_subtract
         48   2F 000022D0 sfColor_toInteger
         49   30 000027D0 sfConvexShape_copy
         50   31 00002880 sfConvexShape_create
         51   32 000028C0 sfConvexShape_destroy
         52   33 00001BF0 sfConvexShape_getFillColor
         53   34 00001C30 sfConvexShape_getGlobalBounds
         54   35 000028F0 sfConvexShape_getInverseTransform
         55   36 00001D50 sfConvexShape_getLocalBounds
         56   37 00001D90 sfConvexShape_getOrigin
         57   38 00001DC0 sfConvexShape_getOutlineColor
         58   39 00001E00 sfConvexShape_getOutlineThickness
         59   3A 00001E10 sfConvexShape_getPoint
         60   3B 00001E40 sfConvexShape_getPointCount
         61   3C 00001E50 sfConvexShape_getPosition
         62   3D 00001E90 sfConvexShape_getRotation
         63   3E 00001EA0 sfConvexShape_getScale
         64   3F 000029D0 sfConvexShape_getTexture
         65   40 00001EE0 sfConvexShape_getTextureRect
         66   41 000029E0 sfConvexShape_getTransform
         67   42 00002010 sfConvexShape_move
         68   43 00002040 sfConvexShape_rotate
         69   44 00002050 sfConvexShape_scale
         70   45 00002080 sfConvexShape_setFillColor
         71   46 000020C0 sfConvexShape_setOrigin
         72   47 000020F0 sfConvexShape_setOutlineColor
         73   48 00002130 sfConvexShape_setOutlineThickness
         74   49 00002AC0 sfConvexShape_setPoint
         75   4A 00002AF0 sfConvexShape_setPointCount
         76   4B 00002150 sfConvexShape_setPosition
         77   4C 00002190 sfConvexShape_setRotation
         78   4D 000021A0 sfConvexShape_setScale
         79   4E 00002B00 sfConvexShape_setTexture
         80   4F 00002210 sfConvexShape_setTextureRect
         81   50 0013201C sfCyan
         82   51 00004190 sfFloatRect_contains
         83   52 000041E0 sfFloatRect_getPosition
         84   53 00004200 sfFloatRect_getSize
         85   54 00004220 sfFloatRect_intersects
         86   55 00003330 sfFont_copy
         87   56 00003490 sfFont_createFromFile
         88   57 000035C0 sfFont_createFromMemory
         89   58 00003680 sfFont_createFromStream
         90   59 00003750 sfFont_destroy
         91   5A 000037C0 sfFont_getBoldKerning
         92   5B 000037E0 sfFont_getGlyph
         93   5C 000038A0 sfFont_getInfo
         94   5D 000038C0 sfFont_getKerning
         95   5E 000038E0 sfFont_getLineSpacing
         96   5F 000038F0 sfFont_getTexture
         97   60 00003950 sfFont_getUnderlinePosition
         98   61 00003960 sfFont_getUnderlineThickness
         99   62 00003970 sfFont_isSmooth
        100   63 00003990 sfFont_setSmooth
        101   64 0013200C sfGreen
        102   65 000039A0 sfImage_copy
        103   66 00003AB0 sfImage_copyImage
        104   67 00003B00 sfImage_create
        105   68 00003B70 sfImage_createFromColor
        106   69 00003BF0 sfImage_createFromFile
        107   6A 00003CF0 sfImage_createFromMemory
        108   6B 00003D80 sfImage_createFromPixels
        109   6C 00003DF0 sfImage_createFromStream
        110   6D 00003E90 sfImage_createMaskFromColor
        111   6E 00003EE0 sfImage_destroy
        112   6F 00003F10 sfImage_flipHorizontally
        113   70 00003F20 sfImage_flipVertically
        114   71 00003F30 sfImage_getPixel
        115   72 00003F70 sfImage_getPixelsPtr
        116   73 00003F80 sfImage_getSize
        117   74 00003FB0 sfImage_saveToFile
        118   75 00004060 sfImage_saveToMemory
        119   76 00004130 sfImage_setPixel
        120   77 000043F0 sfIntRect_contains
        121   78 00004450 sfIntRect_getPosition
        122   79 00004460 sfIntRect_getSize
        123   7A 00004480 sfIntRect_intersects
        124   7B 00132018 sfMagenta
        125   7C 00005810 sfMouse_getPositionRenderWindow
        126   7D 00005850 sfMouse_setPositionRenderWindow
        127   7E 00004620 sfRectangleShape_copy
        128   7F 00004820 sfRectangleShape_create
        129   80 00004860 sfRectangleShape_destroy
        130   81 00001BF0 sfRectangleShape_getFillColor
        131   82 00001C30 sfRectangleShape_getGlobalBounds
        132   83 00004890 sfRectangleShape_getInverseTransform
        133   84 00001D50 sfRectangleShape_getLocalBounds
        134   85 00001D90 sfRectangleShape_getOrigin
        135   86 00001DC0 sfRectangleShape_getOutlineColor
        136   87 00001E00 sfRectangleShape_getOutlineThickness
        137   88 00001E10 sfRectangleShape_getPoint
        138   89 00001E40 sfRectangleShape_getPointCount
        139   8A 00001E50 sfRectangleShape_getPosition
        140   8B 00001E90 sfRectangleShape_getRotation
        141   8C 00001EA0 sfRectangleShape_getScale
        142   8D 00004970 sfRectangleShape_getSize
        143   8E 00004990 sfRectangleShape_getTexture
        144   8F 00001EE0 sfRectangleShape_getTextureRect
        145   90 000049A0 sfRectangleShape_getTransform
        146   91 00002010 sfRectangleShape_move
        147   92 00002040 sfRectangleShape_rotate
        148   93 00002050 sfRectangleShape_scale
        149   94 00002080 sfRectangleShape_setFillColor
        150   95 000020C0 sfRectangleShape_setOrigin
        151   96 000020F0 sfRectangleShape_setOutlineColor
        152   97 00002130 sfRectangleShape_setOutlineThickness
        153   98 00002150 sfRectangleShape_setPosition
        154   99 00002190 sfRectangleShape_setRotation
        155   9A 000021A0 sfRectangleShape_setScale
        156   9B 00004A80 sfRectangleShape_setSize
        157   9C 00004AB0 sfRectangleShape_setTexture
        158   9D 00002210 sfRectangleShape_setTextureRect
        159   9E 00132008 sfRed
        160   9F 00004AF0 sfRenderStates_default
        161   A0 00004C60 sfRenderTexture_clear
        162   A1 00004CA0 sfRenderTexture_create
        163   A2 00004E40 sfRenderTexture_createWithSettings
        164   A3 00005030 sfRenderTexture_destroy
        165   A4 000050B0 sfRenderTexture_display
        166   A5 000050C0 sfRenderTexture_drawCircleShape
        167   A6 000050C0 sfRenderTexture_drawConvexShape
        168   A7 00005130 sfRenderTexture_drawPrimitives
        169   A8 000050C0 sfRenderTexture_drawRectangleShape
        170   A9 000050C0 sfRenderTexture_drawShape
        171   AA 000050C0 sfRenderTexture_drawSprite
        172   AB 000050C0 sfRenderTexture_drawText
        173   AC 000050C0 sfRenderTexture_drawVertexArray
        174   AD 000051A0 sfRenderTexture_drawVertexBuffer
        175   AE 00005210 sfRenderTexture_drawVertexBufferRange
        176   AF 00005280 sfRenderTexture_generateMipmap
        177   B0 000052A0 sfRenderTexture_getDefaultView
        178   B1 000052B0 sfRenderTexture_getMaximumAntialiasingLevel
        179   B2 000052C0 sfRenderTexture_getSize
        180   B3 000052F0 sfRenderTexture_getTexture
        181   B4 00005300 sfRenderTexture_getView
        182   B5 00005310 sfRenderTexture_getViewport
        183   B6 00005360 sfRenderTexture_isRepeated
        184   B7 00005380 sfRenderTexture_isSmooth
        185   B8 000053A0 sfRenderTexture_isSrgb
        186   B9 000053C0 sfRenderTexture_mapCoordsToPixel
        187   BA 00005420 sfRenderTexture_mapPixelToCoords
        188   BB 00005480 sfRenderTexture_popGLStates
        189   BC 00005490 sfRenderTexture_pushGLStates
        190   BD 000054A0 sfRenderTexture_resetGLStates
        191   BE 000054B0 sfRenderTexture_setActive
        192   BF 000054D0 sfRenderTexture_setRepeated
        193   C0 000054E0 sfRenderTexture_setSmooth
        194   C1 000054F0 sfRenderTexture_setView
        195   C2 00005890 sfRenderWindow_capture
        196   C3 00005910 sfRenderWindow_clear
        197   C4 00005950 sfRenderWindow_close
        198   C5 00005960 sfRenderWindow_create
        199   C6 00005C00 sfRenderWindow_createFromHandle
        200   C7 00005DC0 sfRenderWindow_createUnicode
        201   C8 00006020 sfRenderWindow_destroy
        202   C9 00006050 sfRenderWindow_display
        203   CA 00006060 sfRenderWindow_drawCircleShape
        204   CB 00006060 sfRenderWindow_drawConvexShape
        205   CC 000060D0 sfRenderWindow_drawPrimitives
        206   CD 00006060 sfRenderWindow_drawRectangleShape
        207   CE 00006060 sfRenderWindow_drawShape
        208   CF 00006060 sfRenderWindow_drawSprite
        209   D0 00006060 sfRenderWindow_drawText
        210   D1 00006060 sfRenderWindow_drawVertexArray
        211   D2 00006140 sfRenderWindow_drawVertexBuffer
        212   D3 000061B0 sfRenderWindow_drawVertexBufferRange
        213   D4 000052A0 sfRenderWindow_getDefaultView
        214   D5 00006220 sfRenderWindow_getPosition
        215   D6 00006250 sfRenderWindow_getSettings
        216   D7 000062B0 sfRenderWindow_getSize
        217   D8 000062E0 sfRenderWindow_getSystemHandle
        218   D9 00005300 sfRenderWindow_getView
        219   DA 000062F0 sfRenderWindow_getViewport
        220   DB 00006340 sfRenderWindow_hasFocus
        221   DC 00006360 sfRenderWindow_isOpen
        222   DD 00006380 sfRenderWindow_isSrgb
        223   DE 000063A0 sfRenderWindow_mapCoordsToPixel
        224   DF 00006410 sfRenderWindow_mapPixelToCoords
        225   E0 00006470 sfRenderWindow_pollEvent
        226   E1 000064B0 sfRenderWindow_popGLStates
        227   E2 000064C0 sfRenderWindow_pushGLStates
        228   E3 000064D0 sfRenderWindow_requestFocus
        229   E4 000064E0 sfRenderWindow_resetGLStates
        230   E5 000064F0 sfRenderWindow_setActive
        231   E6 00006510 sfRenderWindow_setFramerateLimit
        232   E7 00006520 sfRenderWindow_setIcon
        233   E8 00006530 sfRenderWindow_setJoystickThreshold
        234   E9 00006540 sfRenderWindow_setKeyRepeatEnabled
        235   EA 00006550 sfRenderWindow_setMouseCursor
        236   EB 00006560 sfRenderWindow_setMouseCursorGrabbed
        237   EC 00006570 sfRenderWindow_setMouseCursorVisible
        238   ED 00006580 sfRenderWindow_setPosition
        239   EE 000065A0 sfRenderWindow_setSize
        240   EF 000065C0 sfRenderWindow_setTitle
        241   F0 000066B0 sfRenderWindow_setUnicodeTitle
        242   F1 00006740 sfRenderWindow_setVerticalSyncEnabled
        243   F2 00006750 sfRenderWindow_setView
        244   F3 000067F0 sfRenderWindow_setVisible
        245   F4 00006800 sfRenderWindow_waitEvent
        246   F5 00006890 sfShader_bind
        247   F6 000068A0 sfShader_createFromFile
        248   F7 00006D70 sfShader_createFromMemory
        249   F8 00007240 sfShader_createFromStream
        250   F9 00007430 sfShader_destroy
        251   FA 00007460 sfShader_getNativeHandle
        252   FB 00007470 sfShader_isAvailable
        253   FC 00007490 sfShader_isGeometryAvailable
        254   FD 000074B0 sfShader_setBoolUniform
        255   FE 00007570 sfShader_setBvec2Uniform
        256   FF 00007640 sfShader_setBvec3Uniform
        257  100 00007720 sfShader_setBvec4Uniform
        258  101 00007800 sfShader_setColorParameter
        259  102 000078E0 sfShader_setColorUniform
        260  103 00007960 sfShader_setCurrentTextureParameter
        261  104 00007960 sfShader_setCurrentTextureUniform
        262  105 00007A10 sfShader_setFloat2Parameter
        263  106 00007AE0 sfShader_setFloat3Parameter
        264  107 00007BC0 sfShader_setFloat4Parameter
        265  108 00007CB0 sfShader_setFloatParameter
        266  109 00007CB0 sfShader_setFloatUniform
        267  10A 00007D70 sfShader_setFloatUniformArray
        268  10B 00007E30 sfShader_setIntColorUniform
        269  10C 00007E70 sfShader_setIntUniform
        270  10D 00007F30 sfShader_setIvec2Uniform
        271  10E 00008000 sfShader_setIvec3Uniform
        272  10F 000080E0 sfShader_setIvec4Uniform
        273  110 000081C0 sfShader_setMat3Uniform
        274  111 000082A0 sfShader_setMat3UniformArray
        275  112 00008360 sfShader_setMat4Uniform
        276  113 00008440 sfShader_setMat4UniformArray
        277  114 00008500 sfShader_setTextureParameter
        278  115 000085C0 sfShader_setTextureUniform
        279  116 00008680 sfShader_setTransformParameter
        280  117 000087A0 sfShader_setVec2Uniform
        281  118 00008880 sfShader_setVec2UniformArray
        282  119 00008940 sfShader_setVec3Uniform
        283  11A 00008A20 sfShader_setVec3UniformArray
        284  11B 00008AE0 sfShader_setVec4Uniform
        285  11C 00008BB0 sfShader_setVec4UniformArray
        286  11D 00008C70 sfShader_setVector2Parameter
        287  11E 00008D50 sfShader_setVector3Parameter
        288  11F 00008EE0 sfShape_create
        289  120 00008F60 sfShape_destroy
        290  121 00001BF0 sfShape_getFillColor
        291  122 00001C30 sfShape_getGlobalBounds
        292  123 000028F0 sfShape_getInverseTransform
        293  124 00001D50 sfShape_getLocalBounds
        294  125 00001D90 sfShape_getOrigin
        295  126 00001DC0 sfShape_getOutlineColor
        296  127 00001E00 sfShape_getOutlineThickness
        297  128 00001E10 sfShape_getPoint
        298  129 00001E40 sfShape_getPointCount
        299  12A 00001E50 sfShape_getPosition
        300  12B 00001E90 sfShape_getRotation
        301  12C 00001EA0 sfShape_getScale
        302  12D 000029D0 sfShape_getTexture
        303  12E 00001EE0 sfShape_getTextureRect
        304  12F 000029E0 sfShape_getTransform
        305  130 00002010 sfShape_move
        306  131 00002040 sfShape_rotate
        307  132 00002050 sfShape_scale
        308  133 00002080 sfShape_setFillColor
        309  134 000020C0 sfShape_setOrigin
        310  135 000020F0 sfShape_setOutlineColor
        311  136 00002130 sfShape_setOutlineThickness
        312  137 00002150 sfShape_setPosition
        313  138 00002190 sfShape_setRotation
        314  139 000021A0 sfShape_setScale
        315  13A 00002B00 sfShape_setTexture
        316  13B 00002210 sfShape_setTextureRect
        317  13C 00008F90 sfShape_update
        318  13D 00009000 sfSprite_copy
        319  13E 000091B0 sfSprite_create
        320  13F 000091F0 sfSprite_destroy
        321  140 00009230 sfSprite_getColor
        322  141 00009270 sfSprite_getGlobalBounds
        323  142 000092B0 sfSprite_getInverseTransform
        324  143 00009390 sfSprite_getLocalBounds
        325  144 00001D90 sfSprite_getOrigin
        326  145 00001E50 sfSprite_getPosition
        327  146 00001E90 sfSprite_getRotation
        328  147 00001EA0 sfSprite_getScale
        329  148 000093D0 sfSprite_getTexture
        330  149 000093E0 sfSprite_getTextureRect
        331  14A 00009430 sfSprite_getTransform
        332  14B 00002010 sfSprite_move
        333  14C 00002040 sfSprite_rotate
        334  14D 00002050 sfSprite_scale
        335  14E 00009510 sfSprite_setColor
        336  14F 000020C0 sfSprite_setOrigin
        337  150 00002150 sfSprite_setPosition
        338  151 00002190 sfSprite_setRotation
        339  152 000021A0 sfSprite_setScale
        340  153 00009550 sfSprite_setTexture
        341  154 00009590 sfSprite_setTextureRect
        342  155 000097D0 sfText_copy
        343  156 00009A00 sfText_create
        344  157 00009A70 sfText_destroy
        345  158 00009B50 sfText_findCharacterPos
        346  159 00009B80 sfText_getCharacterSize
        347  15A 00009B90 sfText_getColor
        348  15B 00009B90 sfText_getFillColor
        349  15C 00009BD0 sfText_getFont
        350  15D 00009BE0 sfText_getGlobalBounds
        351  15E 00009C20 sfText_getInverseTransform
        352  15F 00009D00 sfText_getLetterSpacing
        353  160 00009D10 sfText_getLineSpacing
        354  161 00009D20 sfText_getLocalBounds
        355  162 00001D90 sfText_getOrigin
        356  163 00009D60 sfText_getOutlineColor
        357  164 00009DA0 sfText_getOutlineThickness
        358  165 00001E50 sfText_getPosition
        359  166 00001E90 sfText_getRotation
        360  167 00001EA0 sfText_getScale
        361  168 00009DB0 sfText_getString
        362  169 00009F10 sfText_getStyle
        363  16A 00009F20 sfText_getTransform
        364  16B 0000A000 sfText_getUnicodeString
        365  16C 00002010 sfText_move
        366  16D 00002040 sfText_rotate
        367  16E 00002050 sfText_scale
        368  16F 0000A020 sfText_setCharacterSize
        369  170 0000A030 sfText_setColor
        370  171 0000A030 sfText_setFillColor
        371  172 0000A070 sfText_setFont
        372  173 0000A0A0 sfText_setLetterSpacing
        373  174 0000A0B0 sfText_setLineSpacing
        374  175 000020C0 sfText_setOrigin
        375  176 0000A0C0 sfText_setOutlineColor
        376  177 0000A100 sfText_setOutlineThickness
        377  178 00002150 sfText_setPosition
        378  179 00002190 sfText_setRotation
        379  17A 000021A0 sfText_setScale
        380  17B 0000A110 sfText_setString
        381  17C 0000A200 sfText_setStyle
        382  17D 0000A210 sfText_setUnicodeString
        383  17E 0000A2A0 sfTexture_bind
        384  17F 0000A2C0 sfTexture_copy
        385  180 0000A340 sfTexture_copyToImage
        386  181 0000A3C0 sfTexture_create
        387  182 0000A480 sfTexture_createFromFile
        388  183 0000A5F0 sfTexture_createFromImage
        389  184 0000A6E0 sfTexture_createFromMemory
        390  185 0000A7D0 sfTexture_createFromStream
        391  186 0000A8E0 sfTexture_createSrgbFromFile
        392  187 0000AA60 sfTexture_createSrgbFromImage
        393  188 0000AB50 sfTexture_createSrgbFromMemory
        394  189 0000AC40 sfTexture_createSrgbFromStream
        395  18A 0000AD60 sfTexture_destroy
        396  18B 0000ADB0 sfTexture_generateMipmap
        397  18C 0000ADD0 sfTexture_getMaximumSize
        398  18D 0000ADE0 sfTexture_getNativeHandle
        399  18E 0000ADF0 sfTexture_getSize
        400  18F 0000AE20 sfTexture_isRepeated
        401  190 0000AE40 sfTexture_isSmooth
        402  191 0000AE60 sfTexture_isSrgb
        403  192 0000AE80 sfTexture_setRepeated
        404  193 0000AE90 sfTexture_setSmooth
        405  194 0000AEA0 sfTexture_swap
        406  195 0000AEB0 sfTexture_updateFromImage
        407  196 0000AEC0 sfTexture_updateFromPixels
        408  197 0000AED0 sfTexture_updateFromRenderWindow
        409  198 0000AEE0 sfTexture_updateFromTexture
        410  199 0000AED0 sfTexture_updateFromWindow
        411  19A 00006840 sfTouch_getPositionRenderWindow
        412  19B 000E8AB0 sfTransform_Identity
        413  19C 0000AEF0 sfTransform_combine
        414  19D 0000B070 sfTransform_equal
        415  19E 0000B0A0 sfTransform_fromMatrix
        416  19F 0000B100 sfTransform_getInverse
        417  1A0 0000B1F0 sfTransform_getMatrix
        418  1A1 0000B2B0 sfTransform_rotate
        419  1A2 0000B3D0 sfTransform_rotateWithCenter
        420  1A3 0000B500 sfTransform_scale
        421  1A4 0000B620 sfTransform_scaleWithCenter
        422  1A5 0000B770 sfTransform_transformPoint
        423  1A6 0000B850 sfTransform_transformRect
        424  1A7 0000B920 sfTransform_translate
        425  1A8 0000BA40 sfTransformable_copy
        426  1A9 0000BB60 sfTransformable_create
        427  1AA 0000BB90 sfTransformable_destroy
        428  1AB 0000BBC0 sfTransformable_getInverseTransform
        429  1AC 0000BC90 sfTransformable_getOrigin
        430  1AD 0000BCB0 sfTransformable_getPosition
        431  1AE 0000BCD0 sfTransformable_getRotation
        432  1AF 0000BCE0 sfTransformable_getScale
        433  1B0 0000BD00 sfTransformable_getTransform
        434  1B1 0000BDD0 sfTransformable_move
        435  1B2 0000BDF0 sfTransformable_rotate
        436  1B3 0000BE00 sfTransformable_scale
        437  1B4 0000BE20 sfTransformable_setOrigin
        438  1B5 0000BE40 sfTransformable_setPosition
        439  1B6 0000BE60 sfTransformable_setRotation
        440  1B7 0000BE70 sfTransformable_setScale
        441  1B8 001342E0 sfTransparent
        442  1B9 0000BE90 sfVertexArray_append
        443  1BA 0000BED0 sfVertexArray_clear
        444  1BB 0000BEE0 sfVertexArray_copy
        445  1BC 0000C040 sfVertexArray_create
        446  1BD 0000C070 sfVertexArray_destroy
        447  1BE 0000C0B0 sfVertexArray_getBounds
        448  1BF 0000C0F0 sfVertexArray_getPrimitiveType
        449  1C0 0000C100 sfVertexArray_getVertex
        450  1C1 0000C110 sfVertexArray_getVertexCount
        451  1C2 0000C120 sfVertexArray_resize
        452  1C3 0000C130 sfVertexArray_setPrimitiveType
        453  1C4 0000C140 sfVertexBuffer_bind
        454  1C5 0000C150 sfVertexBuffer_copy
        455  1C6 0000C1B0 sfVertexBuffer_create
        456  1C7 0000C250 sfVertexBuffer_destroy
        457  1C8 0000C280 sfVertexBuffer_getNativeHandle
        458  1C9 0000C290 sfVertexBuffer_getPrimitiveType
        459  1CA 0000C2A0 sfVertexBuffer_getUsage
        460  1CB 0000C2B0 sfVertexBuffer_getVertexCount
        461  1CC 0000C2C0 sfVertexBuffer_isAvailable
        462  1CD 0000C2E0 sfVertexBuffer_setPrimitiveType
        463  1CE 0000C2F0 sfVertexBuffer_setUsage
        464  1CF 0000C300 sfVertexBuffer_swap
        465  1D0 0000C310 sfVertexBuffer_update
        466  1D1 0000C330 sfVertexBuffer_updateFromVertexBuffer
        467  1D2 0000C350 sfView_copy
        468  1D3 0000C3F0 sfView_create
        469  1D4 0000C420 sfView_createFromRect
        470  1D5 0000C4A0 sfView_destroy
        471  1D6 0000C4B0 sfView_getCenter
        472  1D7 0000C4D0 sfView_getRotation
        473  1D8 0000BC90 sfView_getSize
        474  1D9 0000C4E0 sfView_getViewport
        475  1DA 0000C530 sfView_move
        476  1DB 0000C550 sfView_reset
        477  1DC 0000C570 sfView_rotate
        478  1DD 0000C580 sfView_setCenter
        479  1DE 0000C5A0 sfView_setRotation
        480  1DF 0000C5B0 sfView_setSize
        481  1E0 0000C5D0 sfView_setViewport
        482  1E1 0000C5F0 sfView_zoom
        483  1E2 00132004 sfWhite
        484  1E3 00132014 sfYellow
  ```
 
</details>

**Comment by massung** [comment]
The currently package was built against CSFML 2.5.1. I have had plans to make a few improvements. As it now seems that 2.6.0 is released, this would be a good time. Thanks for finding this!

**Comment by Weissstreifen** [comment]
I've just played around a bit and the latest SFML mentions sfTexture_setSrgb in the documentation but it's not there anymore. Strange. https://csfml.1l.is/api/Graphics/Texture.html#c.sfTexture_isSrgb 

Removing the mentioned function from the Racket bindings works. But I'm pretty sure there are more changes to the API from 2.5.1 to 2.6.1 than just this function, so sadly it's not a very robust fix. 

@massung are you working on this? I could tackle the issue in a few weeks.

**Comment by massung** [comment]
Sorry, I've been crazy busy with work and life lately. I'd be more than happy to look at - and accept - some PRs. The CSFML library for racket is here: https://github.com/massung/racket-csfml.

There is a PR for the `isSrgb` already, but I haven't accepted it (yet) because everything else is still on 2.5.1. 

What I _want_ to do is get it so that all the CSFML dynamic libraries are installed with the package code at the same time. I got that working with a raylib package I put together a while ago, but ran into issues with CSFML due to libsndfile1.
